<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CafeteriaAdmin</class>
 <widget class="QFrame" name="CafeteriaAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,0,0);
font: 87 8pt &quot;Lato Black&quot;;</string>
   </property>
   <property name="text">
    <string>CHOISIR CATEGORIE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="research_2">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 12pt &quot;Lato&quot;;
color:rgb(0,0,0);
border-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 87 8pt &quot;Lato Black&quot;;</string>
   </property>
   <property name="text">
    <string>RECHERCHE:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,0,0);
font: 87 italic 9pt &quot;Lato Black&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>ALL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FOOD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DRINKS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SNACKS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRINTING</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: transparent;
border-image: url(:/arrow_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="Affichage_produit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>481</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 10pt &quot;Lato Medium&quot;;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>441</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string>ACHAT EN COURS</string>
   </property>
   <widget class="QTableView" name="Affichage_produit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 italic 10pt &quot;Lato Medium&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
font: 87 8pt &quot;Lato Black&quot;;</string>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
font: 87 8pt &quot;Lato Black&quot;;</string>
    </property>
    <property name="text">
     <string>Confirmer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
